### PR TITLE
Yusak

### PR DESCRIPTION
**Pull request by yusakwijaya**
1st commit & 2nd commit 
- focus on initiate scss file and scss loader
- add google font inter
- set base.css with css variable and reset.css
- set main.css for global css, such as container etc
- set navbar on app.vue

3rd commit
- fix hsl on base.css
- update css global at main.css
- update navbar structure in app.vue

4th commit 
- add css variable for border-radius at base.css
- add class btn, btn-primary and badge at main.css
- add accordion component with name accordionSearchResult.vue
- make design for search page

5th commit
- make design for add copy page and linked it

6th commit
- integrate the select dropdown with database: page, platform, product, project
- add repository: page, platform, product

7th commit 
- integrate the search result on search.vue
- update search repository

8th commit
- add word repository
- integrate add copy to save the word to firestore

9th commit
- fix PR

### CODE REVIEW

File: src/assets/base.css
Comment (fnaufalh) [inline src/assets/base.css:1]: remove unused code

File: src/components/AccordionSearchResult.vue
Comment (fnaufalh) [inline src/components/AccordionSearchResult.vue:1]: use symantic name for components if this component is reusable

File: src/components/App.vue [hunk 1]
```diff
@@ -4,46 +4,151 @@ import { RouterLink } from 'vue-router'
...
-  header .wrapper {
-    display: flex;
-    place-items: flex-start;
-    flex-wrap: wrap;
+  p::after {
+    content: '|';
+    padding-left: 1rem;
   }
 }
+// header {
```
Comment (fnaufalh) [inline src/components/App.vue]: remove unused code

File: src/components/Search.vue [hunk 2]
```diff
@@ -1,43 +1,274 @@
...
       console.log('search')
-    }
+    },
+    
+    toggleAccordion() {
+      this.isOpen = !this.isOpen;
+    },
+  },
+  mounted() {
+    // console.log('getheight', this.$refs.blockHeight.clientHeight)
```
Comment (fnaufalh) [inline src/components/Search.vue]: remove unused code

File: src/components/Search.vue [hunk 3]
```diff
@@ -1,43 +1,274 @@
...
       keyword: null,
-      results: []
+      results: [],
+      isOpen: false,
     }
   },
   methods: {
     async searchData() {
       this.results = await searchRepository.search(this.keyword)
       console.log('search')
```
Comment (fnaufalh) [inline src/components/Search.vue]: remove console.log

File: src/components/Search.vue [hunk 4]
```diff
@@ -1,43 +1,274 @@
...
+                  <p>Note</p>
+                  <p>Inactive caused by design change</p>
+                </div>
+              </div>
+            </template>
+          </accordion-search-result>
+        </div>
+      </div>
+
+      <!-- <div class="result-item">
```
Comment (fnaufalh) [inline src/components/Search.vue]: remove unused code

File: src/repositories/SearchRepository.ts [hunk 5]
```diff
@@ -34,8 +40,11 @@ export class SearchRepository {
     const projects = projectSnapshot.docs.map((doc) => {
       return { id: doc.id, ...doc.data() }
     })
-    console.log({ users, roles, projects })
+    const words = wordSnapshot.docs.map((doc) => {
+      return {id: doc.id, ...doc.data()}
+    })
+    console.log({ users, roles, projects, words })
```
Comment (fnaufalh) [inline src/repositories/SearchRepository.ts]: remove log

**Review by fnaufalh — CHANGES_REQUESTED** [review]
Please comment with annotation for features that are under development. don't let functional feature released to production

**Comment by yusakwijaya** [comment]
I've fixed the PR feedback

**Review by fnaufalh — APPROVED** [review]
lgtm 🔥

**Pull request merged**